src/components/DappBrowser/control-panel/ControlPanel.tsx

```diff
@@ -112,7 +112,7 @@ export const ControlPanel = () => {
       const wallet = walletsWithBalancesAndNames[key];
       const filteredAccounts = wallet.addresses.filter(account => account.visible);
       filteredAccounts.forEach(account => {
-        const walletBalance = account.balance === '0.00' ? '0' : account.balance;
+        const walletBalance = Number(account.balance) === 0 ? 0 : account.balance;
```

Comment:
not familiar with the types here, but if `account.balance` can ever be `undefined`, `Number(undefined)` is `NaN`,  `NaN !== 0` so walletBalance will also be `undefined`
```suggestion
        const walletBalance = Number(account.balance) ? account.balance : 0;
```

Reply:
Good catch, thanks